test/integration/rounds.js

```diff
@@ -58,6 +58,15 @@ describe('rounds', () => {
 		return Queries.getAccounts().then(rows => {
 			const accounts = {};
 			_.map(rows, acc => {
+				acc.nameExist = acc.nameexist;
+				acc.u_nameExist = acc.u_nameexist;
+				acc.multiLifetime = acc.multilifetime;
+				acc.u_multiLifetime = acc.u_multilifetime;
+				delete acc.nameexist;
+				delete acc.u_nameexist;
+				delete acc.multilifetime;
+				delete acc.u_multilifetime;
```

Comment:
Yes I would like to have seperate fixtures, but this change is not related to fixtures. Its just fixing up result from the `raw query`. Since `Queries.getAccounts()` on many places as well, so can't actually update the query itself.